src/dom/Struct.ts

```diff
@@ -149,11 +149,7 @@ export class Struct extends Value(
   }
 
   elements(): Value[] {
```

Comment:
Interestingly, our doc comment for `Value#elements` never said anything about de-duping values:

https://github.com/amazon-ion/ion-js/blob/f83fcf0e2c359b52c13ac62db149b0f2ee3ef58b/src/dom/Value.ts#L159-L163

...so now the behavior aligns with its documentation.